collectors/profile.plugin/README.md

```diff
@@ -0,0 +1,5 @@
+You can configure this plugin with the following options:
```

Comment:
Where can the user configure it? Please, take a look in the first paragraph of [apps.plugin](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#configuration) readme.